pages/docs/typescript.ja.mdx

```diff
@@ -34,12 +34,12 @@ const { data } = useSWR(uid, fetcher)
 // `data` は `User | undefined` となります。
 ```
 
-By default, [the error thrown](/docs/error-handling) inside the `fetcher` function has type `any`. The type can also explicitly specified.
+同様に、デフォルトでは、`fetcher`関数内で投げられる[エラー](/docs/error-handling)の型は`any`ですが、型を明示的に指定することができます。
```

Comment:
I've added extra spaces for consistency.

```suggestion
同様に、デフォルトでは、`fetcher` 関数内で投げられる [エラー](/docs/error-handling) の型は `any` ですが、型を明示的に指定することができます。
```

Reply:
Thank you for your suggestion about format.
It seems good, so I commited it.